README.md

```diff
@@ -14,7 +14,7 @@ Using Instabug is as easy as "Get ready, Get set, Go".
 1. <b>Adding Instabug to your dependencies</b> (Getting set)
 
     ```groovy
-        implementation 'com.instabug.library:instabug:4.11.1'
+        implementation 'com.instabug.library:instabug:4.13.1'
```

Comment:
@khataha92 don't forget to update the wiki as well. And it might be a good time to either remove the [Using snapshots](https://github.com/Instabug/android-sample/wiki/Using-snapshots) section or update it (either with generic placeholder or the latest snapshots). Don't you think?